### PR TITLE
L-Space: fuck Github edition

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Stuff for an L-Space thing that will totally exist eventually.
Adds book-wall things, enchanted flying books, kickstool crabs, and an icon for a sentient book thing i'm gonna end up coding eventually

Also gotta remember to force Deity to make the walls drop books when they get broken

control station six 2015-04-14 183030

### CODE REVIEW

File: code/game/objects/effects/decals/Cleanable/misc.dm [hunk 1]
```diff
@@ -163,4 +163,14 @@
 
 /obj/effect/decal/cleanable/soot/New()
 	..()
-	dir = pick(cardinal)
\ No newline at end of file
+	dir = pick(cardinal)
+
+/obj/effect/decal/lspaceclutter
```
Comment (d3athrow) [inline code/game/objects/effects/decals/Cleanable/misc.dm]: Should be a child of the clutter object if there is one.

Comment (NigglyWiggly) [inline code/game/objects/effects/decals/Cleanable/misc.dm]: No specific clutter object beyond the 'generic' one, but I just noticed I forgot to add the 'cleanable' thing to it so lemme fix that real quick

**Comment by DeityLink** [comment]
hm, I might make another sprite for the bookwalls. It feels weird to have them face the front like that.

File: code/game/objects/structures/lspace.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+/obj/structure/lspacewall
+	name = "stack of books"
+	desc = "Large pile of books."
+	icon_state = "lspacewall"
```
Comment (ComicIronic) [inline code/game/objects/structures/lspace.dm]: Is this a valid icon_state on its own? It might cause issues in mapping otherwise.

**Comment by ComicIronic** [comment]
Nice looking content. It would be interesting to see an away mission that used this stuff.

**Comment by NigglyWiggly** [comment]
I was planning on that sort of thing, but I'll probably want to add a bit more content before I try to get it merged again.

**Comment by NigglyWiggly** [comment]
https://github.com/d3athrow/vgstation13/pull/4263

**Pull request closed without merging**